packages/clay-css/src/scss/components/_badges.scss

```diff
@@ -113,7 +113,11 @@
 			starts-with($color, '#') or
 			starts-with($color, '%'),
 		$color,
-		str-insert($color, '.badge-', 1)
+		if(
+			starts-with($color, 'badge'),
```

Comment:
Why do we need to have different logic for beta? Can't we remove this change and just let the old logic insert `.badge-` for beta too?

Reply:
We've moved to using key names that are closer to the selector name. The pattern to use:
```
$map: (
    primary: (),
    secondary: (),
    ...
);
```
is from Bootstrap. The new way should be:
```
$map: (
    btn-primary: (),
    btn-secondary: (),
);
```
These extra steps just provide backward compatibility for Clay 2+.